### PR TITLE
Feat/general better integration tweaks

### PR DESCRIPTION
**Pull request by aviveiros11**
#### Any background context you want to provide?
Addressing feedback from the demo
#### What's this PR do?
Modifies the UI to improve BETTER analysis usability
#### How should this be manually tested?
Check the layout of the configuration and output files are more descriptive and that the min r^2 value is set to 0.6
#### What are the relevant tickets?
#2801, #2802, #2803
#### Screenshots (if appropriate)
Screen Shot 2021-08-11 at 11 08 08 AM
Screen Shot 2021-08-11 at 11 36 58 AM
Screen Shot 2021-08-11 at 11 41 02 AM

### CODE REVIEW

**Comment by RDmitchell** [comment]
@aviveiros11 -- these changes all look great !! Thx !

File: seed/static/seed/partials/analyses.html [hunk 1]
```diff
@@ -13,34 +13,45 @@ <h2><i class="fa fa-bar-chart"></i> {$:: 'Analyses' | translate $}</h2>
...
+                               ng-click="delete_analysis(analysis.id)"></i>
+                        </td>
+                        <td>{$:: analysis.number_of_analysis_property_views $}</td>
+                        <td>{$:: analysis.service $}</td>
+                        <td>
+                            <ul ng-repeat="(config, setting) in analysis.configuration">
+                                <li ng-if="!has_children(setting)">{$:: config $} = {$:: setting $}</li>
+                                <li ng-repeat="(key, value) in setting">{$:: key $} = {$:: value $}</li>
+                            </ul>
+                        </td>
```
Comment (macintoshpie) [inline seed/static/seed/partials/analyses.html:49]: Nice 😎

**Review by macintoshpie — APPROVED** [review]
Looks great, thanks!

File: seed/static/seed/partials/analysis_runs.html [hunk 2]
```diff
@@ -15,8 +15,9 @@
                 <td>{$:: (messages | filter : {'analysis_property_view':view.id})[0]['user_message'] $}</td>
                 <td>
                     <span ng-repeat="file in view.output_files | filter:filter_params:strict">
-                        <a href="{$:: file.file $}" download>{$:: file.content_type $} <span class="fa fa-download"></span></a>
-                        <span ng-repeat-end ng-if="!$last">,&nbsp;</span>
+                        <a ng-if="file['analysis_property_views'].length > 1" href="{$:: file.file $}" download>Portfolio Report <span class="fa fa-download"></span></a>
+                        <a ng-if="file['analysis_property_views'].length == 1" href="{$:: file.file $}" download>Building Report <span class="fa fa-download"></span></a>
+                        <span ng-repeat-end ng-if="!$last">, </span>
```
[the remark this answers is not in the document]
Comment (aviveiros11) [reply seed/static/seed/partials/analysis_runs.html:20]: Yeah, I wonder if the `analysis_output_file` model could have a type awareness field to supply more context?  Eg. building vs portfolio output files?  Would need to noodle on it for a bit.

**Pull request merged**